app/controllers/dashboard/trainer_controller.rb

```diff
@@ -45,4 +23,15 @@ def review_card
   def trainer_params
     params.permit(:user_translation)
   end
+
+  def check_distant(check_result)
+    if check_result[:distance] == 0
+      flash[:notice] = t(:correct_translation_notice)
+    else
+      flash[:alert] = t 'translation_from_misprint_alert',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.